common.py

```diff
@@ -161,7 +161,8 @@ async def convert(self, ctx: commands.Context, argument: str):
 def embed_factory(ctx: commands.Context) -> discord.Embed:
     """Creates an embed with neutral colour and standard footer."""
     embed = discord.Embed(timestamp=datetime.utcnow(), colour=colours.neutral)
-    embed.set_footer(text=str(ctx.author), icon_url=str(ctx.author.avatar_url))
+    if ctx.author and ctx.author.avatar:
```

Comment:
is this related to some new caching behaviour?

Reply:
it's more for mypy